### ISSUE TITLE
Labels to the left of the widget

### ISSUE THREAD

**Issue by kcpevey** [ioam/parambokeh#46]
Is there a way to get labels on the side of widget as opposed to on top of the widget?

**Comment by jbednar** [comment]
No, but there should be.

**Comment by kcpevey** [comment]
And also maybe some column functionality similar to how holoviews handles things with `.col(1)`?

**Comment by jbednar** [comment]
Yes!  We are very ready to dive into issues like this over the coming weeks; there are a lot of things that need doing, and we've never before had the time to address them.  You can see the discussion at https://github.com/ioam/paramnb/issues/42 for some of the things we'd like to do, and for now the main difficulty is just which bit to address first!

**Comment by bacalfa** [comment]
I look forward to this! :) I have four widgets on top of a scatter plot, and if I want to increase its height, there's no space in the screen for the widgets and the plot. So I have to keep scrolling up and down after every selection.
